test/quantization/core/test_quantized_op.py

```diff
@@ -3886,20 +3890,27 @@ def _make_qconv_tensors(
             W = W_scales_tensor.reshape(*W_shape) * (
                 W_init.float() - W_zero_points_tensor.reshape(*W_shape)).float()
             b = X_scale * W_scales_tensor * b_init.float()
+            W_scales_tensor = W_scales_tensor.to(device)
+            W_zero_points_tensor = W_zero_points_tensor.to(device)
         else:
             W = W_scale[0] * (W_init - W_zero_point[0]).float()
             b = X_scale * W_scale[0] * b_init.float()
 
+        # set Tensors to use the correct device
```

Comment:
nit: could these be moved to the constructor?

Reply:
yeah I think so, will do